perceval/backends/core/github.py

```diff
@@ -120,83 +148,133 @@ def fetch(self, from_date=DEFAULT_DATETIME):
 
         from_date = datetime_to_utc(from_date)
 
-        issues_groups = self.client.get_issues(from_date)
+        issues_groups = self.client.get_issues(start=from_date)
 
         for raw_issues in issues_groups:
+            self._push_cache_queue('{ISSUES}')
             self._push_cache_queue(raw_issues)
             self._flush_cache_queue()
             issues = json.loads(raw_issues)
             for issue in issues:
-                for field in ['user', 'assignee']:
+                for field in TARGET_ISSUE_FIELDS:
+                    issue[field + "_data"] = {}
+
                     if issue[field]:
-                        issue[field + "_data"] = self.__get_user(issue[field]['login'])
-                    else:
-                        issue[field + "_data"] = {}
+                        if field == "user":
+                            issue[field + "_data"] = self.__get_user(issue[field]['login'])
+                        elif field == "assignee":
+                            issue[field + "_data"] = self.__get_issue_assignee(issue[field])
+                        elif field == "comments":
+                            issue[field + "_data"] = self.__get_issue_comments(issue["number"])
+                self._push_cache_queue('{}')
+                self._flush_cache_queue()
                 yield issue
 
+    def __fetch_issues_from_cache(self, cache_items):
+        """Fetch issues from cache"""
+
+        raw_content = next(cache_items)
+        issues = (i for i in json.loads(raw_content))
```

Comment:
Why a tuple and not a list?

Reply:
```python
(i for i in json.loads(raw_content))
``` 
should be the pythonic way to return a generator